op-acceptance-tests/tests/flashblocks/flashblocks_stream_test.go

```diff
@@ -51,38 +59,103 @@ func TestFlashblocksStream(gt *testing.T) {
 	logger.Info("Flashblocks stream rate", "rate", flashblocksStreamRateMs)
 
 	// Test all L2 chains in the system
-	for l2Chain, flashblocksBuilderSet := range sys.FlashblocksBuilderSets {
-		_, span = tracer.Start(ctx, "test chain")
-		defer span.End()
-
-		networkName := l2Chain.String()
-		t.Run(fmt.Sprintf("L2_Chain_%s", networkName), func(tt devtest.T) {
-			if len(flashblocksBuilderSet) == 0 {
-				tt.Skip("no flashblocks builders for chain", l2Chain.String())
-			}
+	oprbuilderNode := sys.L2OPRBuilder
+	rollupBoostNode := sys.L2RollupBoost
+	_, span = tracer.Start(ctx, "test chain")
+	defer span.End()
 
-			expectedChainID := l2Chain.ChainID().ToBig()
-			for _, flashblocksBuilderNode := range flashblocksBuilderSet {
-				require.Equal(t, flashblocksBuilderNode.Escape().ChainID().ToBig(), expectedChainID, "flashblocks builder node chain id should match expected chain id")
+	expectedChainID := sys.L2Chain.ChainID().ToBig()
+	require.Equal(t, oprbuilderNode.Escape().ChainID().ToBig(), expectedChainID, "flashblocks builder node chain id should match expected chain id")
+
+	driveViaTestSequencer(t, sys, 3)
+
+	// testFlashblocksStreamRbuilder(t, logger, oprbuilderNode, FlashblocksStreamMode_Leader, flashblocksStreamRateMs)
+
+	// Test the presence / absence of a flashblocks stream operating at a 250ms (configurable via env var FLASHBLOCKS_STREAM_RATE) rate from a flashblocks-websocket-proxy node.
+	// Allow a generous window for first flashblocks to appear.
+	testDuration := time.Duration(int64(flashblocksStreamRateMs*maxExpectedFlashblocks*2)) * time.Millisecond
+	failureTolerance := int(0.15 * float64(maxExpectedFlashblocks))
```

Comment:
I assume `failureTolerance` is the max percent of flashblocks that can be dropped before we fail the test? Comment might be nice since it's not immediately clear when reading top-to-bottom.